Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -846,6 +846,38 @@ class PaymentSheetStandardLPMUITests: PaymentSheetUITestCase {
         let successText = app.staticTexts["Success!"]
         XCTAssertTrue(successText.waitForExistence(timeout: 10.0))
     }
+
+    func testFPXPaymentMethodHasDropdown() throws {
```

Comment:
Hrm I thought I commented on this already, but I don't see it. Is this test useful?  I think FPX is using the same LUXE codepath as other dropdowns and there's nothing unique about it to test here.

Reply:
True, makes sense!